knip.json

```diff
@@ -64,6 +63,7 @@
     "@types/cors",
     "cdktf",
     "cdktf-cli",
+    "constructs",
```

Comment:
remove this and the three cdktf dependencies above so I could see how CI responds from that